### ISSUE TITLE
Hide Error Details

### ISSUE THREAD

**Issue by arknable** [nanovms/ops#1066]
As mentioned on https://github.com/nanovms/ops/issues/1064. I think we need to recheck what error message to display and keep the detail for debug flag.

**Comment by arknable** [comment]
@eyberg Please verify.

**Comment by eyberg** [comment]
in general - many of the clouds have verbose api error output that we should hide in favor of short human readable errors when debug is not turned on; but this is kind of a broad ticket; in general if there is an error, especially in code in ops it should always produce a human readable error for the end-user

**Comment by arknable** [comment]
@eyberg Yes, I'm thinking to modify error log method to something like `log.Error(message string, detail error)` .... we display `message` but hide `detail` by default.
